nixos/modules/services/monitoring/thanos.nix

```
@@ -642,6 +646,10 @@ let
 
     receive = params.common cfg.receive // params.objstore cfg.receive // {
 
+      grpc-compression = mkParam types.str ''
+        Compression algorithm to use for gRPC requests to other clients.
```

```suggestion
        Compression algorithm to use for gRPC requests to other receivers.
```

To be consistent with the Thanos documentation on the query subcommand.
What do you think?